Added lease quota count and rate limit count docs (#22509)
hashicorp/vault: website/content/api-docs/system/rate-limit-quotas.mdx

  concept of roles (such as `/auth/approle/`), this will make the quota restrict login
  requests to that mount that are made with the specified role. The request will fail if
  the auth mount does not have a concept of roles, or `path` is not an auth mount.

> - `inheritable` `(bool: false)` - If set to `true` on a quota where `path` is set to a namespace,
>   all child namespaces of the specified path namespace will inherit this quota. `inheritable` cannot be set to
>   true if the `path` does not specify a namespace. All root namespace quotas are inheritable by default.
>   All other namespace quotas are not inheritable by default.

Reviewer: ```suggestion
- `inheritable` `(bool: false)` - If set to `true` on a quota where `path` is set to a namespace,
  all child namespaces of the specified path namespace will inherit the quota. The `inheritable`
  parameter cannot be set to `true` if the `path` does not specify a namespace. Only quotas
  associated with the root namespace quotas are inheritable by default.
```

Same suggestions as for the other file
